@ndlib/gatsby-theme-marble/src/utils/mapStandardJson/data/allLanguageCodes.js

```diff
@@ -0,0 +1,3411 @@
+    alpha2: 'su',
+    'alpha3-b': 'sun',
+    'alpha3-t': null,
+  },
+  {
+    English: 'Susu',
+    French: 'soussou',
+    alpha2: null,
+    'alpha3-b': 'sus',
+    'alpha3-t': null,
+  },
+  {
+    English: 'Sumerian',
+    French: 'sumérien',
+    alpha2: null,
+    'alpha3-b': 'sux',
+    'alpha3-t': null,
+  },
+  {
+    English: 'Swahili',
+    French: 'swahili',
+    alpha2: 'sw',
+    'alpha3-b': 'swa',
+    'alpha3-t': null,
+  },
+  {
+    English: 'Swedish',
+    French: 'suédois',
+    alpha2: 'sv',
+    'alpha3-b': 'swe',
+    'alpha3-t': null,
+  },
+  {
+    English: 'Classical Syriac',
+    French: 'syriaque classique',
+    alpha2: null,
+    'alpha3-b': 'syc',
+    'alpha3-t': null,
+  },
+  {
+    English: 'Syriac',
+    French: 'syriaque',
+    alpha2: null,
+    'alpha3-b': 'syr',
+    'alpha3-t': null,
+  },
+  {
+    English: 'Tahitian',
+    French: 'tahitien',
+    alpha2: 'ty',
+    'alpha3-b': 'tah',
+    'alpha3-t': null,
+  },
+  {
+    English: 'Tai languages',
+    French: 'tai, langues',
+    alpha2: null,
+    'alpha3-b': 'tai',
+    'alpha3-t': null,
+  },
+  {
+    English: 'Tamil',
+    French: 'tamoul',
+    alpha2: 'ta',
+    'alpha3-b': 'tam',
+    'alpha3-t': null,
+  },
+  {
+    English: 'Tatar',
+    French: 'tatar',
+    alpha2: 'tt',
+    'alpha3-b': 'tat',
+    'alpha3-t': null,
+  },
+  {
+    English: 'Telugu',
+    French: 'télougou',
+    alpha2: 'te',
+    'alpha3-b': 'tel',
+    'alpha3-t': null,
+  },
+  {
+    English: 'Timne',
+    French: 'temne',
+    alpha2: null,
+    'alpha3-b': 'tem',
+    'alpha3-t': null,
+  },
+  {
+    English: 'Tereno',
+    French: 'tereno',
+    alpha2: null,
+    'alpha3-b': 'ter',
+    'alpha3-t': null,
+  },
+  {
+    English: 'Tetum',
+    French: 'tetum',
+    alpha2: null,
+    'alpha3-b': 'tet',
+    'alpha3-t': null,
+  },
+  {
+    English: 'Tajik',
+    French: 'tadjik',
+    alpha2: 'tg',
+    'alpha3-b': 'tgk',
+    'alpha3-t': null,
+  },
+  {
+    English: 'Tagalog',
+    French: 'tagalog',
+    alpha2: 'tl',
+    'alpha3-b': 'tgl',
+    'alpha3-t': null,
+  },
+  {
+    English: 'Thai',
+    French: 'thaï',
+    alpha2: 'th',
+    'alpha3-b': 'tha',
+    'alpha3-t': null,
+  },
+  {
+    English: 'Tibetan',
+    French: 'tibétain',
+    alpha2: 'bo',
+    'alpha3-b': 'tib',
+    'alpha3-t': 'bod',
+  },
+  {
+    English: 'Tigre',
+    French: 'tigré',
+    alpha2: null,
+    'alpha3-b': 'tig',
+    'alpha3-t': null,
+  },
+  {
+    English: 'Tigrinya',
+    French: 'tigrigna',
+    alpha2: 'ti',
+    'alpha3-b': 'tir',
+    'alpha3-t': null,
+  },
+  {
+    English: 'Tiv',
+    French: 'tiv',
+    alpha2: null,
+    'alpha3-b': 'tiv',
+    'alpha3-t': null,
+  },
+  {
+    English: 'Tokelau',
+    French: 'tokelau',
+    alpha2: null,
+    'alpha3-b': 'tkl',
+    'alpha3-t': null,
+  },
+  {
+    English: 'Klingon; tlhIngan-Hol',
+    French: 'klingon',
+    alpha2: null,
+    'alpha3-b': 'tlh',
+    'alpha3-t': null,
```

Comment:
![XSzjB9C](https://user-images.githubusercontent.com/416559/87977427-b6df0700-ca9c-11ea-90b4-3b6749c0dd68.jpg)

Reply:
you never know what will be in the catalog.